api/organisations/tasks.py

```diff
@@ -255,21 +252,35 @@ def charge_for_api_call_count_overages():
     ):
         subscription_cache = organisation.subscription_information_cache
         api_usage = get_current_api_usage(organisation.id, "30d")
-        api_usage_ratio = api_usage / subscription_cache.allowed_30d_api_calls
 
-        if api_usage_ratio < 1.0:
-            logger.warning("API Usage does not match API Notification")
-            continue
+        api_billing = (
+            OrganisationAPIBilling.objects.filter(
+                billed_at__gte=subscription_cache.current_billing_term_starts_at
+            )
+            .order_by("-billed_at")
+            .first()
```

Comment:
There should only ever be one of these records, right? Do we care about the `order_by`? Should we assert that there is only one?

Reply:
No these `OrganisationAPIBilling` objects are created each time someone has been billed. So if in a given month a customer has multiple overages each time a new billing. That said, I made a mistake with the logic around the billed amount since the overages are only the latest billings when it should be a sum of the billings within the month. I'll update this pr with a fix when I can.